Tools/update-wiitdb.sh

```diff
@@ -1,26 +1,21 @@
 #!/bin/bash
-# Script to update the bundled WiiTDBs and remove duplicate entries.
+# Script to update the bundled WiiTDBs.
 
 set -eu
-ENGLISH_DB_NAME="Data/Sys/wiitdb-en.txt"
-ENGLISH_TMP_DB_NAME="Data/Sys/wiitdb-en.tmp"
-TIMESTAMP_PATTERN="TITLES = "
-
-echo "Downloading WiiTDB (en)..."
-curl -s "http://www.gametdb.com/wiitdb.txt?LANG=EN" | sort > "$ENGLISH_TMP_DB_NAME"
 
 dbs=()
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=DE de")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=ES es")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=FR fr")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=IT it")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=JA ja")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=KO ko")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=NL nl")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=PT pt")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=RU ru")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=ZHCN zh_CN")
-dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=ZHTW zh_TW")
+dbs+=("http://www.gametdb.com/wiitdb.txt?LANG=EN en")
```

Comment:
Nope. UNIQUE means "only return non-English titles", so using it with English doesn't make sense :P